nucypher/blockchain/eth/token.py

```diff
@@ -531,13 +533,19 @@ class WorkTracker:
     INTERVAL_FLOOR = 60 * 15  # fifteen minutes
     INTERVAL_CEIL = 60 * 180  # three hours
 
+    ALLOWED_DEVIATION = 0.5  # i.e., up to +50% from the expected confirmation time
+
     def __init__(self, worker, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
         self.log = Logger('stake-tracker')
         self.worker = worker
         self.staking_agent = self.worker.staking_agent
 
+        self.gas_strategy = self.staking_agent.blockchain.gas_strategy
+        expected_time = EXPECTED_CONFIRMATION_TIME_IN_SECONDS[self.gas_strategy]
+        self.max_confirmation_time = expected_time * (1 + self.ALLOWED_DEVIATION)
```

Comment:
I assume this is why glacial became too risky hehe.

Reply:
It's too cheap to be helpful in a reasonable time frame.  Perhaps combined with an escalating strategy...